### PR TITLE
migrate to slash syntax

### PR DESCRIPTION
**Pull request by jtjeferreira**
In preparation for sbt 2.x, migrate to slash syntax

### CODE REVIEW

**Review by muuki88 — APPROVED** [review]
Amazing @jtjeferreira :heart: :heart:

**Comment by muuki88** [comment]
I think you added slashes in a few places where they aren't correct :D 
https://github.com/sbt/sbt-native-packager/actions/runs/11256140432/job/31297541037?pr=1636#step:5:351

**Comment by jtjeferreira** [comment]
> I think you added slashes in a few places where they aren't correct :D [sbt/sbt-native-packager/actions/runs/11256140432/job/31297541037?pr=1636#step:5:351](https://github.com/sbt/sbt-native-packager/actions/runs/11256140432/job/31297541037?pr=1636#step:5:351)

I might have added slashes too the wrong places, but I don't see where... I noticied that github actions are now using ubuntu-24 instead of ubuntu-22 like in the 3 weeks ago ubuntu-22.04. I will investigate

File: src/main/scala/com/typesafe/sbt/packager/universal/UniversalPlugin.scala [hunk 1]
```diff
@@ -90,19 +90,19 @@ object UniversalPlugin extends AutoPlugin {
...
-      universalArchiveOptions in (Universal, packageXzTarball) := Seq("-pcvf"),
-      universalArchiveOptions in (UniversalDocs, packageZipTarball) := Seq("-pcvf"),
-      universalArchiveOptions in (UniversalDocs, packageXzTarball) := Seq("-pcvf"),
-      universalArchiveOptions in (UniversalSrc, packageZipTarball) := Seq("-pcvf"),
-      universalArchiveOptions in (UniversalSrc, packageXzTarball) := Seq("-pcvf")
+      Universal / packageZipTarball / universalArchiveOptions := Seq("-pcvf"),
+      Universal / packageXzTarball / universalArchiveOptions := Seq("-pcvf"),
+      UniversalDocs / packageZipTarball / universalArchiveOptions := Seq("-pcvf"),
+      UniversalDocs / packageXzTarball / universalArchiveOptions := Seq("-pcvf"),
+      UniversalDocs / packageXzTarball / universalArchiveOptions := Seq("-pcvf"),
```
Comment (jtjeferreira) [inline src/main/scala/com/typesafe/sbt/packager/universal/UniversalPlugin.scala:104]:
```suggestion
      UniversalSrc / packageXzTarball / universalArchiveOptions := Seq("-pcvf"),
```

File: src/main/scala/com/typesafe/sbt/packager/universal/UniversalPlugin.scala [hunk 2]
```diff
@@ -90,19 +90,19 @@ object UniversalPlugin extends AutoPlugin {
...
-      universalArchiveOptions in (Universal, packageXzTarball) := Seq("-pcvf"),
-      universalArchiveOptions in (UniversalDocs, packageZipTarball) := Seq("-pcvf"),
-      universalArchiveOptions in (UniversalDocs, packageXzTarball) := Seq("-pcvf"),
-      universalArchiveOptions in (UniversalSrc, packageZipTarball) := Seq("-pcvf"),
-      universalArchiveOptions in (UniversalSrc, packageXzTarball) := Seq("-pcvf")
+      Universal / packageZipTarball / universalArchiveOptions := Seq("-pcvf"),
+      Universal / packageXzTarball / universalArchiveOptions := Seq("-pcvf"),
+      UniversalDocs / packageZipTarball / universalArchiveOptions := Seq("-pcvf"),
+      UniversalDocs / packageXzTarball / universalArchiveOptions := Seq("-pcvf"),
+      UniversalSrc / packageXzTarball / universalArchiveOptions := Seq("-pcvf"),
```
Comment (jtjeferreira) [inline src/main/scala/com/typesafe/sbt/packager/universal/UniversalPlugin.scala:104]:
```suggestion
      UniversalSrc / packageZipTarball / universalArchiveOptions := Seq("-pcvf"),
```

File: src/main/scala/com/typesafe/sbt/packager/archetypes/JavaAppPackaging.scala [hunk 3]
```diff
@@ -1,6 +1,6 @@
 package com.typesafe.sbt.packager.archetypes
 
-import sbt._
+import sbt.{given, _}
```
Comment (jtjeferreira) [inline src/main/scala/com/typesafe/sbt/packager/archetypes/JavaAppPackaging.scala:3]: leftover from sbt 2.x migration
```suggestion
import sbt._
```

**Comment by jtjeferreira** [comment]
The scripted tests still contain deprecated syntax like `universal:packageBin`, happy to fix it here or in another PR

**Review by muuki88 — APPROVED** [review]

**Comment by muuki88** [comment]
Thanks a lot @jtjeferreira for this huge contribution. Especially because it's not fancy stuff, but highly necessary cleanup work :pray: :pray:

**Pull request merged**